File: PspDataPresenter.App/data-presenter-app/src/app/_core/models/notification.model.ts
import { NotificationCancelType } from "../enums/notification-cancel-type.enum";
import { NotificationStartType } from "../enums/notification-start-type.enum";
import { NotificationType } from "../enums/notification-type.enum";
import { RoomModel } from "./room.model";

export interface NotificationModel {
    id: number;
    created: Date | string;
    started: Date | string;
    ended: Date | string | null;
    type: NotificationType;
    meeting1Number: number;
    meeting2Number: number | null;
    meeting1Time1: Date | string | null;
    meeting1Time2: Date | string | null;
    meeting2Time: Date | string | null;
    startType: NotificationStartType;
    cancelType: NotificationCancelType;
    text: string;
    room: RoomModel;
    timestamp: Date | string;
}

File: PspDataPresenter.Core/Models/NotificationModel.cs
using System;

namespace PspDataPresenter.Core
{
    public class NotificationModel
    {
        public int Id { get; set; }
        public DateTime Created { get; set; }
        public DateTime Started { get; set; }
        public DateTime? Ended { get; set; }
        public string Type { get; set; }
        public int? Meeting1Number { get; set; }
        public int? Meeting2Number { get; set; }
        public DateTime? Meeting1Time1 { get; set; }
        public DateTime? Meeting1Time2 { get; set; }
        public DateTime? Meeting2Time { get; set; }
        public string StartType { get; set; }
        public string CancelType { get; set; }
        public string Text { get; set; }
        public RoomModel Room { get; set; }
        public DateTime Timestamp { get; set; }
    }
}